ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/OpenAPIParameterMapper.java

```diff
@@ -68,34 +67,39 @@ public class OpenAPIParameterMapper {
     private final FunctionDefinitionNode functionDefinitionNode;
     private final OperationAdaptor operationAdaptor;
     private final Map<String, String> apidocs;
-    private final List<OpenAPIMapperDiagnostic> errors;
+    private final List<OpenAPIMapperDiagnostic> diagnostics = new ArrayList<>();
```

Comment:
Does `diagnostics` hold anything other than errors ? If so shall we rename the `getErrors` method to `getDiagnostics` ? or else we should rename this field to `errors`.